authors/apps/authentication/test/test_authentication.py

```diff
@@ -44,82 +47,3 @@ def test_user_registration_wrong_email_format(self):
         user = self.generate_user('athena', 'athenmail', 'password@user')
         response = self.client.post('/api/users/', user, format='json')
         self.assertEqual(response.status_code, status.HTTP_400_BAD_REQUEST)
-
-    def test_user_login(self):
```

Comment:
Any reasons why you completely removed these tests?

Reply:
I discovered that Kasule had also written the same tests while working on the authentication with jwt feature, so it was like duplicate code

Reply:
But diff shows you deleted them. Meaning they were already part of the version you branched off (I'm assuming you branched of `develop`). You should therefore leave them as they are or refactor if necessary.

Reply:
Ya I branched of develop and created them and then deleted them after
The code base has no doc strings. But I can put them pack hope ti will not cause conflicts again

Reply:
You could just rebase against `develop` and resolve the conflicts before pushing.

Reply:
alright